browsers/firefox_android.json

```diff
@@ -511,27 +511,29 @@
         "89": {
           "release_date": "2021-06-01",
           "release_notes": "https://developer.mozilla.org/docs/Mozilla/Firefox/Releases/89",
-          "status": "current",
+          "status": "retired",
           "engine": "Gecko",
           "engine_version": "89"
         },
         "90": {
           "release_date": "2021-07-13",
           "release_notes": "https://developer.mozilla.org/docs/Mozilla/Firefox/Releases/90",
-          "status": "beta",
+          "status": "current",
           "engine": "Gecko",
           "engine_version": "90"
         },
         "91": {
           "release_date": "2021-08-10",
           "release_notes": "https://developer.mozilla.org/docs/Mozilla/Firefox/Releases/91",
-          "status": "nightly",
+          "status": "beta",
           "engine": "Gecko",
           "engine_version": "91"
         },
         "92": {
           "release_date": "2021-09-07",
           "status": "planned",
+          "release_notes": "https://developer.mozilla.org/docs/Mozilla/Firefox/Releases/92",
+          "status": "nightly",
```

Comment:
Note that this added a second "status" key. Surprisingly that didn't fail any tests, but anyway it was the only difference to https://github.com/mdn/browser-compat-data/pull/11511.